lib/ansible/module_utils/homebrew.py

```diff
@@ -0,0 +1,245 @@
+# (c) 2013, Andrew Dunham <andrew@du.nham.ca>
+# (c) 2013, Daniel Jaouen <dcj24@cornell.edu>
+# (c) 2015-2017, Indrajit Raychaudhuri <irc+code@indrajit.com>
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
```

Comment:
We've talked about this on IRC.  @indrajitr will attempt to contact all of the original authors of the code to get approval for re-licensing, and if that fails, I believe the plan is to re-write all the code in a "cleanroom implementation", instead of re-licensing.